### DISCUSSION TITLE
Completely white screen

### DISCUSSION THREAD

**Discussion by DavidHornDesign** [statamic/cms#7268] [Troubleshooting]
Hi - looking for some tips on how I might debug this situation:

- 3 or 4 days ago I had a local Statamic site that was loading fine; blank starter kit, simple TailwindCSS layout, had created a few collections, blueprints, taxonomies, etc. all working fine.
- on revisiting the site this week, all I see is a completely blank screen. The SSL cert loads, and there are no errors on the screen or in console, just completely blank. Same for /cp
- honestly, I know: the first question would be 'what has changed?' ... and I cannot think of anything that's changed. I run LocalWP which I know can sometimes cause a conflict, but all of that - and all associated processes - are closed down.
- I'm using Valet and that starts / stops just fine. Valet secure has generated the SSL.
- npm run watch & npm run dev both seem to work just fine
- and I can create a whole new install of Statamic with no problems ... so, something has obviously just gone wrong with the original site, and I just don't have the process / understanding of how to debug.

I'm just feeling a bit lost! I've come from WordPress where I had a process to follow for debugging problems ... and I don't have a similar process for Statamic. 

Any guidance appreciated. Thank you!

David

**Comment by duncanmcclean** [comment]
Is there any HTML being returned or not?

**Reply by DavidHornDesign** [reply]
No, nothing. Console shows an empty body tag inside an otherwise empty html tag.

**Reply by duncanmcclean** [reply]
Very weird...

If you run 'php artisan serve' it'll run a separate web server. Does it return the same thing or different?

Just trying to rule out Valet being the problem.

**Reply by DavidHornDesign** [reply]
Ahhh ... now it feels like we're getting somewhere, thank you Duncan. So I run the separate server and go to the URL given and it shows an error which looks like it can't find the TW CSS file. 
![CleanShot 2022-12-23 at 18 56 24](https://user-images.githubusercontent.com/60608816/209394551-2dfc4826-d065-4e91-8d0a-a9b427adcc52.jpg)

**Reply by DavidHornDesign** [reply]
is this just me with something as basic as a misdirecting file reference?

**Reply by duncanmcclean** [reply]
Try running 'npm run dev' & reload the page... (you need to compile the CSS & JS, otherwise the files won't load)

**Reply by DavidHornDesign** [reply]
Perfect Duncan; thank you so much for your help - all looks good. So, to help me with further development, is the following all correct:

 - the problem, somehow, was Valet - because ditching valet and running the 'php artisan serve' command fixed it
 - I don't need to run Valet at all ... I was doing so because it seemed like an easy way to create a solid environmemnt - I liked how easy it was to create an SSL, and - coming from MAMP - the 'valet park' command just seemed like magic. But I needn't use it - the 'artisan serve' command does the job to create a server.
 - the sort of error screen I got above - which is really helpful - is what I should expect if something has gone wrong with Statamic / my set up; the white screen / lack of any HTML output indicated (to you) that the problem was more likely with Valet than anything else.

If that's all the case, then great - it's helping me with my debug process. I've been on a few Statamic sites and really love it ... but this threw me; it just seemed so weird.

All good - I really appreciate your help, especially as I'm sure you're supposed to be on holiday. Thank you.

**Reply by duncanmcclean** [reply]
I mean, Valet should really be working. It's better than 'artisan serve'.

I'd maybe try uninstalling it, then reinstalling it to see if that helps.

Haha, yeah, I'm on "holiday" but I spend way too much time refreshing GitHub on my phone 😂

**Reply by DavidHornDesign** [reply]
(uninstalling & re-installing Valet did the trick. Instructions for uninstalling Valet found here: https://github.com/laravel/valet/issues/341#issuecomment-287048987)
